apis/swagger.yml

```diff
@@ -1837,8 +1837,14 @@ definitions:
               type: "string"
           OomScoreAdj:
             type: "integer"
-            description: "An integer value containing the score given to the container in order to tune OOM killer preferences."
-            example: 500
+            description: |
+                An integer value containing the score given to the container in order to tune OOM killer preferences.
+                The range is in [-1000, 1000].
+            type: "integer"
+            format: "int"
+            x-nullable: true
```

Comment:
I do not think this should be ` x-nullable: true`, since it would be a little bit incompatible with moby 1.12.6's API.

See https://github.com/moby/moby/blob/v1.12.6/vendor/src/github.com/docker/engine-api/types/container/host_config.go#L300

So it should be `x-nullable: false`